periodic_job_test.go

```diff
@@ -62,7 +63,7 @@ func TestPeriodicJobBundle(t *testing.T) {
 
 		periodicJob := NewPeriodicJob(
 			PeriodicInterval(15*time.Minute),
-			func() (JobArgs, *InsertOpts) {
+			func() (river.JobArgs, *river.InsertOpts) {
```

Comment:
Can you revert this diff so the file doesn't need a named import for its own package? Seems like that'll fix CI issues.